framework/src/modules/interoperability/mainchain/module.ts

```diff
@@ -299,60 +288,65 @@ export class MainchainInteroperabilityModule extends BaseInteroperabilityModule
 				throw new Error(`ownChainNonce must be positive if chainInfos is not empty.`);
```

Comment:
Question about line 277: Do we need to check that we are in mainchain?? Isn't it assumed that this part is executed only on mainchain?

Reply:
I see you removed this check, so I assume it was not needed. Plz verify (cc. @ishantiw ).